src/components/modules/LogoutButton/LogoutButton.view.vue

```diff
@@ -31,3 +31,11 @@ export default {
   },
 };
 </script>
+
+<style lang="scss" scoped>
+.logout-button {
+  position: absolute;
+  right: 25px;
```

Comment:
Position styles should be defined on the parent level, because component can't know something about its future position, except of `fixed` components